### ISSUE TITLE
When shared externally to element, does not display the sharing process and results

### ISSUE THREAD

**Issue by Simonmxy** [vector-im/element-android#7825]
### Steps to reproduce

When sharing a picture or a video from other app like Google Photos into element room,the sharing interface shown correctly and sharing successful,but the room didnt show the photo or video and uploading process,only a successful mark.
This happened since 1.5.12.

### Outcome

Element 1.5.11 is fine.

### Your phone model

Pixel 6 Pro

### Operating system version

Android 13

### Application version and app store

Element 1.5.13 from here

### Will you send logs?

No

### Are you willing to provide a PR?

No

**Comment by YamatoRyou** [comment]
Might be similar to #6380

**Comment by Simonmxy** [comment]
> 

No,not that one,Sorry for didn't explain clearly, The bug I described will only occur when sharing. When sharing pictures or videos from the outside to the Element room, the sharing interface will appear correctly, but after pressing send menu, the room interface should have the photo and the sending progress bar, and finally display the sharing Success, but since version 1.5.12, there is no media and process bar,only a successful mark.
